lxd/daemon.go

```diff
@@ -146,6 +147,9 @@ type Daemon struct {
 	// Cluster.
 	serverName string
 
+	// Servers UUID from file.
```

Comment:
> // Server's or // Server

Reply:
Thanks, fixed also in the comment below.